src/main/webapp/app/shared/user-settings/ide-preferences/ide-settings.scss

```diff
@@ -0,0 +1,32 @@
+h3 {
+    font-weight: bold;
+    text-underline-offset: 3px;
+}
+
+dt {
+    font-size: large;
+    font-weight: 500;
+}
+
+dd {
+    font-size: larger;
+    color: gray;
+}
+
+span {
+    color: gray;
+    font-size: smaller;
+}
+
+.btn-default {
+    transition: background-color 0.3s ease;
+}
+
+.btn-default:hover {
+    background-color: rgba(0, 0, 0, 0.05);
+    border: 1px solid #999;
```

Comment:
Please use variables for the colors `#999` and rgba(...)

Reply:
done